src/views/article.vue

```diff
@@ -1,69 +1,100 @@
-                     :class="{'reverse-image': i%2 !== 0}">
+                <div class="image-box" v-if="article.chapters[i].image"
+                     :style="getImageStyle(article.chapters[i])" :class="{'reverse-image': i%2 !== 0}">
                     <img class="image-box-deco desktop-only" src="../assets/groups/vector-for-blocks.svg" alt="">
                 </div>
                 <div class="text-box">
                     <img class="text-box-deco desktop-only" src="../assets/groups/paragraph-vector.svg" alt="">
                     <h2 class="text-box-title">{{ chapter.subtitle }}</h2>
-                    <p class="text-box-text" v-html="chapter.paragraph">
-                        {{ chapter.paragraph }}
-                    </p>
+                    <p class="text-box-text" v-html="getParsedText(chapter.paragraph)"></p>
                 </div>
             </div>
         </div>
 
         <!-- FOOTER DECO -->
-        <img class="deco-footer" src="../assets/mobile-image-deco.svg" alt="">
+        <img class="deco-footer" src="../assets/deco/mobile-image-deco.svg" alt="">
     </div>
 </template>
 
 <script>
+import {mapState} from 'vuex';
+
 export default {
     name: 'article-component',
     data() {
         return {
-            wikiArticles: [],
+            article: null,
             articleName: '',
             chapterNumber: 0,
             showInput: true,
         };
     },
+    computed: {
+        ...mapState(['gameData']),
+    },
     watch: {
         $route() {
             this.openArticle();
         },
     },
     methods: {
         openArticle() {
-            this.wikiArticles = [];
             this.category = this.$route.params.category;
             this.articleName = this.$route.params.id;
             import(`../data/articles/${this.category}/${this.articleName}.json`).then((e) => {
-                this.wikiArticles = e;
+                this.article = e;
+            });
+        },
+        getParsedText(text) {
+            if (!text.includes('{{')) return text;
+            // extract data keys and add data from gameData
+            const fragments = text.split('{{');
+            let parsedText = '';
+            if (fragments.length % 2 === 1) parsedText += fragments.shift();
+            fragments.forEach(fragment => {
+                if (fragment.includes('}}')) {
+                    const subFragment = fragment.split('}}');
+                    const dataKey = subFragment[0];
+                    const subKeys = dataKey.split('.');
+                    let data = this.gameData;
+                    for (let key of subKeys) {
+                        key = key.trim();
+                        data = data[key] ? data[key] : 'INVALID_DATA_KEY';
+                    }
+                    parsedText += data;
+                    if (subFragment[1]) parsedText += subFragment[1];
+                } else parsedText += fragment;
             });
+            return parsedText;
+        },
```

Comment:
Woah, nice one! At first when I saw you using the ``{{ data }}`` syntax in a json-file I thought: "How is this supposed to work?". Now I see how.